streaming/src/main/scala/io/gearpump/streaming/task/TaskActor.scala

```diff
@@ -281,6 +259,7 @@ class TaskActor(
         queue.add(msg)
         handler()
       case None =>
+        LOG.error(s"Task $taskId drop message $msg")
```

Comment:
There is one problem, when there is a upstream zombie task keeping sending message, it will flood the log file. 

Maybe we should find some solution which meet:
1. indicate there is unexpected error happening.
2. avoid flooding the log file.